CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 1.2.0 - 11 May 2020
+### Added
+* Semantic highlighting support. [#1840](https://github.com/microsoft/vscode-docker/issues/1840)
+* Help and Feedback pane in explorer view. [#1893](https://github.com/microsoft/vscode-docker/issues/1893)
+* Docker Context pane in explorer view. [#1844](https://github.com/microsoft/vscode-docker/issues/1844)
+* Images can be pulled from the images list. [#1313](https://github.com/microsoft/vscode-docker/issues/1313)
+* Containers can be grouped by docker-compose project name. [#215](https://github.com/microsoft/vscode-docker/issues/215), [#1846](https://github.com/microsoft/vscode-docker/issues/1846)
+* A new setting, `docker.dockerodeOptions`, allowing any options to be provided to Dockerode. [#1459](https://github.com/microsoft/vscode-docker/issues/1459)
+
+### Changed
+* Any file named `Dockerfile.*` is now recognized as a Dockerfile. [#1907](https://github.com/microsoft/vscode-docker/issues/1907)
```

Comment:
What a time to be alive...

Reply:
All this time we spent worrying if not-real-Dockerfiles like `Dockerfile.js` was a problem, and it ended up being 1/20th as common as real Dockerfiles with random extensions like `Dockerfile.amd64`.